modules/KalturaSupport/components/dualScreen/imagePlayer.js

```diff
@@ -100,6 +102,19 @@
 				)
 			);
 		},
+		syncByReachedCuePoints : function()
+		{
+			var _this = this;
+			if (_this.cuePointsManager) {
+				var cuePointsReachedResult = _this.cuePointsManager.getCuePointsReached();
+				if (!_this.cuePointsReached(cuePointsReachedResult)) {
+					// when user seek/press play we need to handle scenario that no relevant cue points has reached and thus we need to clear the image shown.
+					_this.sync(null);
```

Comment:
If we didn't handle any cuepoints, it indicates that we should remove any visible image